tools/fork-network/src/cli.rs

```diff
@@ -480,9 +512,10 @@ impl ForkNetworkCommand {
         self.make_and_write_genesis(
             genesis_time,
             protocol_version,
+            target_shard_layout,
             epoch_length,
             num_seats,
-            block_height,
+            flat_head.height + 1,
```

Comment:
Can you clarify why we need to add `+1`?

Reply:
I honestly don't know.. I bet it doesn't really matter too much. But here I was just making sure to keep the old behavior because in the current code, we do `let block_height = block_height + 1;` before writing it to `FORK_TOOL_BLOCK_HEIGHT`